src/tests/inventory-checking-copy.spec.js

```diff
@@ -0,0 +1,59 @@
+import { expect } from '@playwright/test';
+import { test } from '../fixtures/base';
+import { users } from '../test-data/users';
+
+const { username, password } = users.standardUser;
+
+let initialProducts = [];
+let selectedProducts = [];
+let cartProducts = [];
+
+test.describe('Inventory checking copy', () => {
+    test.beforeEach(async (/** @type {{ app: import('../pages/Application').Application }} */{ app }) => {
+        await test.step('Open login page', async () => {
+            await app.login.navigate();
+        });
+
+        await test.step('Login as a standard user role', async () => {
+            await app.login.performLogin(username, password);
+        });
+
+        initialProducts = await test.step('Get initial data for sorting', () => app.inventory.getProductData());
+
+        await test.step('Checking products count', async () => {
+            expect(initialProducts.length).toBeGreaterThan(2);
+        });
+
+        selectedProducts = initialProducts.slice(0, 3);
```

Comment:
Not really random products. They are always the same. 
It's something just to think about. With randomiser you'll have a little bit more fun)